src/vivarium_gates_iv_iron/components/observers.py

```diff
@@ -43,6 +43,23 @@ def _wrapped_passthrough(*args, **kwargs):
 
 
 class ResultsStratifier(ResultsStratifier_):
+    def setup(self, builder: Builder) -> None:
+        super().setup(builder)
+
+        original_age_bins = self.age_bins
+        is_too_young = original_age_bins['age_start'] < 10
+        is_too_old = original_age_bins['age_end'] > 60
+        original_age_bins = original_age_bins[~(is_too_young) & ~(is_too_old)]
+
+        new_age_bins = pd.DataFrame(
+            data={'age_start': [0.0, 60.0],
+                  'age_end': [10.0, 130.0],
+                  'age_group_name': ['0_to_10', '60_plus']}
```

Comment:
We shouldn't track simulants after they age out. You should remove the 60 plus bin